Bonsai.Tinkerforge/BrickletDeviceNameConverter.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ComponentModel;
+using Bonsai.Expressions;
+using Tinkerforge;
+using System.Threading;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Bonsai.Tinkerforge
+{
+    internal class BrickletDeviceNameConverter : TypeConverter
+    {
+        public Dictionary<string, TinkerforgeHelpers.DeviceData> devices;
+        //private Stopwatch stopwatch;
+
+        public override bool GetStandardValuesSupported(ITypeDescriptorContext context)
+        {
+            return true;
+        }
+
+        public override bool GetStandardValuesExclusive(ITypeDescriptorContext context)
+        {
+            return true;
```

Comment:
We used to have this in drop-down converters, however this prevents users from manually typing values in the property, which can be annoying in situations where enumeration fails, etc, so probably best to remove this.